RetailCoder.VBE/RubberduckGuid.cs

```diff
@@ -24,5 +27,9 @@ public static class RubberduckGuid
         public const string IFakeGuid = "69E194DF" + UnitTestingGuidspace;                          // added for 2.0.14
         public const string IVerifyGuid = "69E194E0" + UnitTestingGuidspace;                        // added for 2.0.14
         public const string IStubGuid = "69E194E1" + UnitTestingGuidspace;                          // added for 2.0.14
+
+        // Enum Guids:
+        public const string DeclarationTypeGuid = "3E077C17-5678-3605-8449-FEABE42C9725";
+        public const string AccessibilityGuid = "6BD869B8-C7C6-3970-8ED4-AC74DD2EBA92";
```

Comment:
I'm thinking the VBA/COM API should have its own guidspace too.. thoughts?

Reply:
Yes, *eventually*. Not in this PR, however, because before we can, we need to clean up the mess we made on every users' computer, and that means we need a good system for COM registration (which this PR doesn't even promise. All we have is the data extraction but nothing actually registering... *yet*).